Embrace Unity iOS Interface/EmbraceManager.swift

```diff
@@ -0,0 +1,494 @@
+    }
+    
+    static func clearUserIdentifier() {
+        Embrace.client?.metadata.userIdentifier = nil
+    }
+    
+    static func addBreadCrumb(event: String) {
+        Embrace.client?.add(event: .breadcrumb(event))
+    }
+    
+    static func setUsername(userName: String) {
+        Embrace.client?.metadata.userName = userName
+    }
+    
+    static func clearUsername() {
+        Embrace.client?.metadata.userName = nil
+    }
+    
+    static func setUserEmail(userEmail: String) {
+        Embrace.client?.metadata.userEmail = userEmail;
+    }
+    
+    static func clearUserEmail() {
+        Embrace.client?.metadata.userEmail = nil
+    }
+    
+    static func addUserPersona(persona: String) -> Bool {
+        do {
+            try Embrace.client?.metadata.add(persona: persona, lifespan: .session)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func clearUserPersona(persona: String) -> Bool {
+        do {
+            try Embrace.client?.metadata.remove(persona: PersonaTag(persona), lifespan: .session)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func clearAllUserPersonas() -> Bool {
+        do {
+            try Embrace.client?.metadata.removeAllPersonas()
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func addResource(key: String, value: String, lifespan: MetadataLifespan) -> Bool {
+        do {
+            try Embrace.client?.metadata.addResource(
+                key: key, value: value, lifespan: lifespan)
+            return true
+        } catch let error {
+            
+        }
+        
+        return false
+    }
+    
+    static func addSessionProperty(key: String, value: String, permanent: Bool) -> Bool {
+        do {
+            let lifespan: MetadataLifespan = permanent ? .permanent : .session
+            try Embrace.client?.metadata.addProperty(key: key, value: value, lifespan: lifespan)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func removeSessionProperty(key: String) -> Bool {
+        do {
+            try Embrace.client?.metadata.removeProperty(key: key, lifespan: .permanent)
+            try Embrace.client?.metadata.removeProperty(key: key, lifespan: .session)
+            return true
+        } catch let error {
+            return false
+        }
+    }
+    
+    static func logMessageWithSeverityAndProperties(
+        message: String,
+        severity: String,
+        properties: [String: String]) {
+            Embrace.client?.log(message,
+                severity: convertStringToLogSeverity(from: severity),
+                attributes: properties
+            )
+    }
+    
+    static func setUserAsPayer() -> Bool {
+        do {
+            try Embrace.client?.metadata.add(persona: "payer")
```

Comment:
There's a shorthand for this in the iOS public api:
```suggestion
            try Embrace.client?.metadata.add(persona: .payer)
```